domain-services/ifix-master-data-service/src/main/java/org/egov/util/GovernmentConstants.java

```diff
@@ -4,8 +4,12 @@
     public static final String USER_INFO = "USER_INFO";
     public static final String GOVERNMENT_ID = "GOVERNMENT_ID";
     public static final String GOVERNMENT_NAME = "GOVERNMENT_NAME";
+    public static final String TENANT_ID = "TENANT_ID";
```

Comment:
If this file is going to include constants apart from Government then change the file name to MasterDataConstants instead of GovernmentConstants. (Do this in a separate PR.

Reply:
I have moved constant names in master constant file name.